packages/battery/example/pubspec.yaml

```diff
@@ -15,4 +15,4 @@ flutter:
   uses-material-design: true
 
 environment:
-  flutter: ">=1.9.1+hotfix.4 <2.0.0"
+  flutter: ">=1.6.7 <2.0.0"
```

Comment:
nit: Does this match the wider pubspec? It may be easier to just delete this entirely.

Reply:
good point, removed.